squad/frontend/templates/squad/_regressions_popover.html

```diff
@@ -0,0 +1,49 @@
+<script>
+{% for status in statuses %}
+    {% if status.regressions is not None %}
+        $('#popperRegressions' + {{ status.build.id }}).popover({
+            placement: 'right',
+            trigger: 'hover',
+            container: 'body',
+            html: true,
+            title: 'Regressions',
+            content: function () {
+                return $('#regressions' + {{ status.build.id }}).html();
+            }
+        });
+        $('#popperRegressions' + {{ status.build.id }} + "_last").popover({
+            placement: 'right',
+            trigger: 'hover',
+            container: 'body',
+            html: true,
+            title: 'Regressions',
+            content: function () {
+                return $('#regressions' + {{ status.build.id }}).html();
+            }
+        });
+    {% endif %}
+    {% if status.fixes is not None %}
+        $('#popperFixes' + {{ status.build.id }}).popover({
+            placement: 'right',
+            trigger: 'hover',
+            container: 'body',
+            html: true,
+            title: 'Fixes',
+            content: function () {
+                return $('#fixes' + {{ status.build.id }}).html();
+            }
+        });
+        $('#popperFixes' + {{ status.build.id }} + "_last").popover({
+            placement: 'right',
+            trigger: 'hover',
+            container: 'body',
+            html: true,
+            title: 'Fixes',
+            content: function () {
+                return $('#fixes' + {{ status.build.id }}).html();
+            }
+        });
+    {% endif %}
+{% endfor %}
+</script>
```

Comment:
this can be simplified by giving classes `popover-regressions` and `popover-fixes` to the buttons, adding "Regressions" and "Fixes" to a `title` attribute, hen doing something like this:

```javascript
jQuery(function($) {
    $('.popover-regressions, .popover-fixes').popover({
            placement: 'right',
            trigger: 'hover',
            container: 'body',
            html: true,
            title: function() { return $(this).attr('title') },
            content: function() { return $(this).parent().find('.hidden').html() }
    })
})
```

Reply:
I think I got this working. PR updated.